docs/swagger.yaml

```diff
@@ -85,85 +85,6 @@ definitions:
           key2: value2
         type: object
     type: object
-  broadcast.SubmittedTx:
-    properties:
-      blockHash:
-        description: BlockHash is the hash of the block where the transaction was
-          included.
-        type: string
-      blockHeight:
-        description: BlockHeight is the height of the block where the transaction
-          was included.
-        type: integer
-      extraInfo:
-        description: ExtraInfo provides extra information for given transaction.
-        type: string
-      merklePath:
-        description: MerklePath is the Merkle path used to calculate Merkle root of
-          the block in which the transaction was included.
-        type: string
-      status:
-        description: Status is the status of the response.
-        type: integer
-      timestamp:
-        description: Timestamp is the timestamp of the block where the transaction
-          was included.
-        type: string
-      title:
-        description: Title is the title of the response.
-        type: string
-      txStatus:
-        allOf:
-        - $ref: '#/definitions/broadcast.TxStatus'
-        description: TxStatus is the status of the transaction.
-      txid:
-        description: TxID is the transaction id.
-        type: string
-    type: object
-  broadcast.TxStatus:
-    enum:
-    - UNKNOWN
-    - QUEUED
-    - RECEIVED
-    - STORED
-    - ANNOUNCED_TO_NETWORK
-    - REQUESTED_BY_NETWORK
-    - SENT_TO_NETWORK
-    - ACCEPTED_BY_NETWORK
-    - SEEN_ON_NETWORK
-    - MINED
-    - SEEN_IN_ORPHAN_MEMPOOL
-    - CONFIRMED
-    - REJECTED
-    type: string
-    x-enum-comments:
```

Comment:
Is it good if this code will be removed?

Reply:
it is removed automatically, I don't know how to regenerate it -> tried with --parsedDependency option and a normal swagger generation 🤷